server.js

```diff
@@ -32,20 +32,20 @@ const dbClientManager = require('./src/koop/dbClientManager');
 const koop = new Koop({});
 
 // Determine the authorization strategy to use; see https://koopjs.github.io/docs/usage/authorization for more info.
-if (config.auth && config.auth.enabled) {
+if (config.auth && config.auth.plugin && config.auth.enabled !== false) {
```

Comment:
This previously required `"enabled":true` to be in the config. I'm assuming the point of this is so that multiple auth configs can be present but only one is enabled. But most of the time, I'd expect there only to be a single auth config, in which case it's simpler to assume that the auth is enabled unless it's explicitly disabled.

Reply:
That seems reasonable.